index.html

```diff
@@ -38,6 +38,106 @@ <h2 class="connect">LET'S CONNECT</h2>
+                            <p class="fa-solid fa-circle fa-2xs"></p>
+                            <p class="year">2015</p>
+                        </div>
+                        <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="language-container">
+                            <li class="lang">html</li>
+                            <li class="lang">css</li>
+                            <li class="lang">javascript</li>
+                        </ul>
+                        <button class="see-project">See Project</button>
+                </div>
+                <div class="cards card2">
+                    <img src="images/Snapshoot-Portfolio-(1).svg" alt="Snapshot Portfolio">
+                    <h3 class="project-name">Multi Post Stories</h3>
+                        <div class="key-text"> 
+                            <p class="canopy">CANOPY</p>
+                            <p class="fa-solid fa-circle fa-2xs"><p>
+                            <p class="position">Back end dev</p>
+                            <p class="fa-solid fa-circle fa-2xs"></p>
+                            <p class="year">2015</p>
+                        </div>
+                        <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="language-container">
+                            <li class="lang">html</li>
+                            <li class="lang">css</li>
+                            <li class="lang">javascript</li>
+                        </ul>
+                        <button class="see-project">See Project</button>
+                </div>
+                <div class="cards card 3">
+                    <img src="images/Snapshoot-Portfolio-(2).svg" alt="Snapshot Portfolio">
+                    <h3 class="project-name">Tonic</h3>
+                        <div class="key-text"> 
+                            <p class="canopy">CANOPY</p>
+                            <p class="fa-solid fa-circle fa-2xs"></p>
+                            <p class="position">Back end dev</p>
+                            <p class="fa-solid fa-circle fa-2xs"></p>
+                            <p class="year">2015</p>
+                        </div>
+                        <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="language-container">
+                            <li class="lang">html</li>
+                            <li class="lang">css</li>
+                            <li class="lang">javascript</li>
+                        </ul>
+                        <button class="see-project">See Project</button>
+                </div>            
+            </div>
+        </section>
+
+        <section class="about">
+            <h1 class="name">About Myself</h1>
```

Comment:
- [ ] [Optional] There should be only one `<h1>` tag on a page for semantic and SEO reasons. Kindly replace `<h1>` tag other than used for your name in the first section with hx tags (h2. .... h6).